src/Ring-Core/RGBehavior.class.st

```diff
@@ -222,6 +222,16 @@ RGBehavior >> asYetUnclassifiedProtocolName [
 	^ Protocol unclassified
 ]
 
+{ #category : #accessing }
+RGBehavior >> buildMicroDownUsing: aMicMicrodownTextualBuilder withComment: aString [ 
```

Comment:
I wonder if this is ok here or this should be an extension method.

Reply:
Ok, added as extension methods in the Microdown repository.